app/helpers/concepts_helper.rb

```diff
@@ -132,15 +132,14 @@ def render_concepts_by_dates(auto_click: false)
     tmp = {}
     tmp[first_year] = first_month_concepts
     @concepts_year_month = tmp.merge(@concepts_year_month)
-
+    selected_id = @concepts.first.id
```

Comment:
this does not work, because `@concepts` does change on each new page. So you will have the same original issue.